Traceback (most recent call last):
  File "/kunden/homepages/8/d657042007/htdocs/CodeSet/private_server/guard/CELL-46/Q-33/46-33.py", line 5, in <module>
    now = now.year()
TypeError: 'int' object is not callable
